### PR TITLE
#2808: Fix possible NPE in AbstractQuickFixTest

### PR DESCRIPTION
**Pull request by imhotep82**
See: https://github.com/eclipse/xtext/issues/2808

### CODE REVIEW

File: org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.java
Comment (imhotep82) [inline org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.java:208]: this could have lead to an NPE

**Comment by cdietrich** [comment]
@imhotep82 can you please rebase on the latest main so that the build can work.

**Comment by imhotep82** [comment]
Sure thing

File: org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.java
Comment (szarnekow) [inline org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.java:75]: Could you also please keep the location of these methods below `getResult`?

Comment (imhotep82) [reply org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.java:75]: sure

**Comment by cdietrich** [comment]
@szarnekow @imhotep82 what is the state of this pr?

**Comment by imhotep82** [comment]
> @szarnekow @imhotep82 what is the state of this pr?

impl is done, waiting for approval

**Review by szarnekow — APPROVED** [review]

**Comment by szarnekow** [comment]
> > @szarnekow @imhotep82 what is the state of this pr?
> 
> impl is done, waiting for approval

My bad, I forgot to correct click sequence. Done. I'll merge.

**Pull request merged**
